examples/main.jsonnet

```diff
@@ -64,8 +64,15 @@ local apiWithTLS = api + api.withTLS {
   config+:: {
     tls+: {
       certFile: './tmp/certs/server.pem',
-      privateKeyFile: './tmp/certs/ca.pem',
-      clientCAFile: './tmp/certs/server.key',
+      privateKeyFile: './tmp/certs/server.key',
+    },
+  },
+};
+
+local withClientTLS = apiWithTLS + api.withClientTLS {
```

Comment:
Since this is both TLS and client TLS, maybe consider naming this local `withMTLS`

Reply:
Done